package.json

```diff
@@ -14,8 +14,8 @@
     "format": "prettier --write \"src/**/*.ts\"",
```

Comment:
So, are we using `prettier` or no?

Reply:
Sorry, I shoulda go to this earlier, but no we're not using it.